tests/performance-test/performance-test.sh

```diff
@@ -126,24 +139,12 @@ delete_es_events(){
     local esPod=$(oc get elasticsearch elasticsearch -o jsonpath='{.status.pods.master.ready[0]}')
     local secretPath="/etc/elasticsearch/secret/"
     local esQuery=$(cat <<EOF
-        {
-            "query" : {
-                "bool": {
-                    "must": {
-                        "match_phrase": {
-                            "labels.instance": {
-                                "query": "saf-perftest-notify-*"
-                            }
-                        }
-                    }
-                }
-            }
-        }
+        { "query" : { "bool": { "must": { "match_phrase": { "labels.instance": { "query": "saf-perftest-notify-*" } } } } } }
 EOF
 )
     printf "\n*** Deleting ES Events ***\n"
 
-    local response=$(oc exec "$esPod" -- curl -k --silent --trace --cacert "${secretPath}admin-ca" --cert \
+    local response=$(oc exec "$esPod" -- curl --silent --cacert "${secretPath}admin-ca" --cert \
```

Comment:
:+1: When I noticed this it was because I saw a file called `--cacert` on my ES pod.

Reply:
Nice catch!